lib/bounces.js

```diff
@@ -59,7 +59,13 @@ module.exports = function (log, error) {
       }
       return P.each(recipients, function (recipient) {
         var email = recipient.emailAddress
-        log.info({ op: 'handleBounce', email: email, bounce: !!message.bounce })
+        log.info({
+          op: 'handleBounce',
+          email: email,
+          bounce: !!message.bounce,
```

Comment:
One minor thing that I noticed is that you don't log the `bounce.bounceType`? This worthwhile?